easylist/easylist_adservers.txt

```diff
@@ -15986,6 +15986,7 @@
 ||interclick.com^$third-party
 ||interesting.cc^$third-party
 ||intergi.com^$third-party
+||intergient.com^$third-party
```

Comment:
I don't see any reasons to set the $third-party there are no frontend to the domain

PS. I'm surprised this wasn't in the list already...

```bash
git log -L '/intergient.com/,+1:source/tracking/wildcard.list'
commit 98d4d5ef9a1843176061ede57a36745698cc3f1c
Author: Spirillen <pdns@protonmail.com>
Date:   Tue Feb 25 04:23:03 2020 +0100

    Deleted `addlvr.com` as expired by @Pyfunceble
    
    Signed-off-by: Spirillen <pdns@protonmail.com>

diff --git a/source/tracking/wildcard.list b/source/tracking/wildcard.list
--- a/source/tracking/wildcard.list
+++ b/source/tracking/wildcard.list
@@ -0,0 +1094,1 @@
+intergient.com
```

Reply:
My bad @spirillen ! Fixed !

Reply:
bad is a good song by Michael Jackson.... :smiley: